ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -51,7 +51,6 @@ const checkFilePreviewInFileAttachment = (component) => {
 const deleteSelectedFiles = (component, fileNames) => {
     cy.get(component).then(() => {
         fileNames.forEach((fileName) => {
-            cy.get(".cmp-adaptiveform-fileinput__filedelete").should('have.attr', 'role', 'button');
```

Comment:
There needs to be test cases for all the version of file input component ? Can you create test suite for v1, v2 and v3.

Also wrt changes done in each version, test cases should reflect that as part of test spec

Reply:
Implemented!